src/fuse.js

```diff
@@ -337,6 +337,67 @@
   };
 
   /**
+   * Dynamically add an item or a list of items to the list
+   * @param {String|Object} to_add The item or list of items to added
+   * @public
+   */
+  Fuse.prototype.add = function(to_add) {
+    Array.prototype.push.apply(this.list, Utils.isArray(to_add) ? to_add : [to_add]);
+  };
+
+  /**
+   * Dynamically remove an item or a list of items to the list
+   * If the list contains strings, items can be removed by value or by index;
+   * If the list contains objects, items can only be removed by id.
+   * @param {String|Number|Array} to_remove The item or list of items to be removed
+   * @public
+   */
+  Fuse.prototype.remove = function(to_remove) {
+    to_remove = Utils.isArray(to_remove) ? to_remove : [to_remove];
+    for (var i = 0, len = to_remove.length; i < len; i++) {
+      Fuse.prototype.remove.apply(to_remove[i]);
+    }
+  };
+
+  /**
+   * Dynamically remove a single item from the list
+   * @private
+   */
+  Fuse.prototype._remove = function(to_remove) {
```

Comment:
Perhaps consider moving this to a function outside of `Fuse.prototype`, since you'll probably be calling with with `apply` anyway.